ta/asym_cipher_perf/include/ta_asym_cipher_perf.h

```diff
@@ -0,0 +1,86 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2015, Linaro Limited
+ * All rights reserved.
+ */
+
+#ifndef TA_ASYM_CIPHER_PERF_H
+#define TA_ASYM_CIPHER_PERF_H

```

Comment:
By the way, this TA does more than just cipher (sign/verify). Rename to _ta_asymm_crypto_perf.h_?